src/ui/ListItemNodeView.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+
+import {Node} from 'prosemirror-model';
+import {Decoration, EditorView} from 'prosemirror-view';
+import {MARK_TEXT_COLOR, MARK_FONT_SIZE} from '../MarkNames';
+
+// This implements the `NodeView` interface
+// https://prosemirror.net/docs/ref/#view.NodeView
+class ListItemNodeView {
+  // This implements the `NodeView` interface
+  // The outer DOM node that represents the list item element.
+  dom: HTMLLIElement;
+
+  // This implements the `NodeView` interface.
+  // The DOM node that should hold the node's content.
+  contentDOM: HTMLLIElement;
+
+  _nodeUpdated: ?Node;
+
+  constructor(
+    node: Node,
+    editorView: EditorView,
+    getPos: () => number,
+    decorations: Array<Decoration>
+  ) {
+    const dom = document.createElement('li');
+    this.dom = dom;
+    this.contentDOM = dom;
+    this._updateDOM(node);
+  }
+
+  // This implements the `NodeView` interface.
+  update(node: Node, decorations: Array<Decoration>): boolean {
+    return this._updateDOM(node);
```

Comment:
some of the our other `NodeView`s call `super`, do we need to here?

Reply:
Good question.

Those who call `super` extends `CustomNodeView` which allows subclass to define the React View to render the UI.

In this Class, no React View is needed, so it simply defines the minimum APIs required to render the list element.